src/ui_session_interface.rs

```diff
@@ -1023,6 +1023,13 @@ impl<T: InvokeUiSession> Session<T> {
         if true == force_relay {
             self.lc.write().unwrap().force_relay = true;
         }
+        if !user_session_id.is_empty() {
+            let mut config = self.load_config();
+            config
+                .options
+                .insert("selected_user_session_id".to_string(), user_session_id);
+            self.save_config(config);
+        }
```

Comment:
still peer option, but works. The original meaning is not to store runtime variables in file , using the same method as force_relay.